frontend/src/Homepage.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from "styled-components";
+import {Link} from "react-router-dom";
+
+
+export default function Homepage() {
+    return <>
+        <StyledHeader>
+            <h1>Dein Baby Tagebuch</h1>
+        </StyledHeader>
+        <main>
+            <StyledDiv>
```

Comment:
here you can simplify the code and remove the divs, maybe for future use you can add more "sections" and display them together